src/style.css

```diff
@@ -1,3 +1,74 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+
+  /* variables */
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)